src/plugins/kibana/server/routes/api/ingest/register_post.js

```diff
@@ -73,8 +74,13 @@ export function registerPost(server) {
       delete indexPattern.id;
 
       const mappings = createMappingsFromPatternFields(indexPattern.fields);
-      indexPattern.fields = initDefaultFieldProps(indexPattern.fields);
 
+      const metaFields = _.get(config, 'metaFields.userValue', config.metaFields.value);
+      const indexPatternMetaFields = _.map(metaFields, name => {
+        return {name};
+      });
```

Comment:
ooo didn't know about that parenthesis trick, I usually prefer the implicit return myself.